cmd/monaco/download/auth.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * @license
+ * Copyright 2025 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package download
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/secret"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest"
+)
+
+type auth struct {
```

Comment:
This commit I'm not too sure about, as I think it should go further: As the `auth` type is only used in `downloadCmdOptions`, why not include the fields in that type directly and place the logic there? Or with similar code:
https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/363b8742e179e42347a2e01e24f9355754f5a7e0/cmd/monaco/download/download_configs.go#L128-L143